flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -284,6 +284,15 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
       }
       userInstance.others = others
     }
+    (user["data"] as? Map<String, Any?>)?.let { data ->
+      val others = mutableMapOf<String, String>()
+      for ((key, value) in data.entries) {
+        if (value != null) {
+          others[key] = value.toString()
+        }
+      }
+      userInstance.data = others
+    }
```

Comment:
Why is this a compile time error, even though data exists? https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/protocol/User.java

Reply:
Not sure, try `setData`? can you compile locally?

Reply:
No, `setData` also doesn't work and I can't compile locally :/

Reply:
Upgrade the Android dep to `6.5.0-beta.3`, this field was added recently.
Or keep it as it is and merge both maps to `others`.